packages/labs/react/src/create-component.ts

```diff
@@ -237,6 +237,9 @@ export const createComponent = <I extends HTMLElement, E extends Events = {}>(
           // React does *not* handle `className` for custom elements so
           // coerce it to `class` so it's handled correctly.
           props[k === 'className' ? 'class' : k] = v;
+          if (v === false && this._element?.hasAttribute(k)) {
```

Comment:
This needs a comment here and the approach should be briefly described in the PR description. In particular, we need to understand how React works for properties like this and what the expected behavior of doing this generally will be.

Reply:
I think this may not be what we want, for example, for an input like `draggable={false}`.

Reply:
I was looking for a react behavior that matches what we need in the DOM.
It might not be conventional but setting `draggable={null}` has the same behavior as `draggable={false}` in react-land with the benefit of it also working in the DOM. But as `null` value isn't represented in the html.

So the following:
```
root.render(<div draggable={false} hidden={false}>
  Produces draggable="false" attribute and no hidden attribute.
</div>);
```

produces:
```
<div draggable="false">Produces draggable="false" attribute and no hidden attribute.</div>
```

While
```
root.render(<div draggable={null} hidden={null}>
  Produces no attribute and no hidden attribute.
</div>);
```

produces:
```
<div>Produces no draggable attribute and no hidden attribute.</div>
```

Reply:
There might be some kind of `null` watch for props in React. But I don't think that that's the case.
What is more likely going on here is react is setting the value `null` for an attribute on `div` and removing the attribute is how the _browser_ works.

For me, that matches the behavior we need with the least about of bashing the constructor.